### PR TITLE
feature - repo cleanup

### PR DESCRIPTION
**Pull request by filgaia**
**Features**
- Fix `dist` path in `.gitignore`
- Fix `yarn build` script
- Fix `yarn test` and skip failing ones
- Force node version to **16**
- Add `clean` script to `package.json`
- Add `globals`to `eslintrc` config files with new `vue3` coding style
- Run `yarn lint` and upload autofixes

### CODE REVIEW

File: packages/card/src/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import UiCard from './ui-card.component.vue'
 import UiCardText from './ui-card-text.component.vue'
 
-UiCard.TAG_NAME = 'uiCard'
-UiCardText.TAG_NAME = 'uiCardText'
+// UiCard.TAG_NAME = 'uiCard'
```
Comment (jimmymadrigal) [inline packages/card/src/index.ts:4]: go ahead and remove this lines completely

File: packages/input/src/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import UiInput from './ui-input.component.vue'
 
-UiInput.TAG_NAME = 'uiInput'
+// UiInput.TAG_NAME = 'uiInput'
```
Comment (jimmymadrigal) [inline packages/input/src/index.ts:3]: go ahead and remove this lines completely

File: packages/pagination/src/index.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import UiPagination from './ui-pagination.component.vue'
 import UiPaginationItem from './ui-pagination-item.component.vue'
 
-UiPagination.TAG_NAME = 'uiPagination'
-UiPaginationItem.TAG_NAME = 'uiPaginationItem'
+// UiPagination.TAG_NAME = 'uiPagination'
```
Comment (jimmymadrigal) [inline packages/pagination/src/index.ts:4]: go ahead and remove this lines completely

File: packages/core/src/utils/config.ts [hunk 4]
```diff
@@ -246,7 +246,7 @@ export function useReactiveResponsiveConfig(
 }
 
 export function setComponentThemeConfig(
-  cmp: { TAG_NAME: string } | string,
+  cmp: { TAG_NAME: string } | string | any,
```
Comment (jimmymadrigal) [inline packages/core/src/utils/config.ts:249]: why any here?

**Review by jimmymadrigal — CHANGES_REQUESTED** [review]

File: packages/core/src/utils/config.ts (hunk 4 above)
Comment (filgaia) [reply packages/core/src/utils/config.ts:249]: Linter complaining and didn't find a good model match

**Pull request merged**